src/modules/likes.js

```diff
@@ -0,0 +1,49 @@
+const getLikes = async () => {
+  try {
+    const response = await fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/MgbGOFTaDhTVdgSX8BJW/likes');
```

Comment:
- Persisting and retrieving record(s) with the [Involvement API](https://www.notion.so/Involvement-API-869e60b5ad104603aa6db59e08150270) well implemented! 👌